packages/sdk/tsup.config.ts

```diff
@@ -7,7 +7,7 @@ export default defineConfig([
     format: ['esm', 'cjs'],
     dts: true,
     sourcemap: true,
-    splitting: true,
+    splitting: false,
```

Comment:
The sdk is not part of the integrations. It can split the bundle if it wants to.

Reply:
Done! Please review and let me know if any further changes are required.